crates/bevy_sprite/src/mesh2d/material.rs

```diff
@@ -484,16 +485,17 @@ pub fn queue_material2d_meshes<M: Material2d>(
 
             match material_2d.properties.alpha_mode {
                 AlphaMode2d::Opaque => {
-                    opaque_phase.add(Opaque2d {
-                        entity: *visible_entity,
-                        draw_function: draw_opaque_2d,
+                    let bin_key = Opaque2dBinKey {
                         pipeline: pipeline_id,
-                        // Front-to-back ordering
-                        sort_key: -FloatOrd(mesh_z + material_2d.properties.depth_bias),
```

Comment:
is this technical breaking since `depth_bias` can no longer be used to prevent z-fighting? the performance trade-off seems obviously worth it since just changing the mesh's z is probably not a big deal but just wanted to check

Reply:
Well, it's not breaking in the sense that the feature doesn't currently exist outside of main so we aren't breaking any contracts. Also, that's what 3d does right now. I would like for this to still be doable in both 2d and 3d, but this should be done in a separate PR that updates both with an alternate solutionl